src/exprs.c

```diff
@@ -999,8 +1008,9 @@ Obj             EvalListTildeExpr (
 **  '[ [1], ~[1] ]'  requires that the value of  one subexpression is entered
 **  into the list value before the next subexpression is evaluated.
 */
-Obj ListExpr1 (
-    Expr                expr )
+ALWAYS_INLINE Obj ListExpr1 (
```

Comment:
Why `ALWAYS_INLINE`?

Reply:
Will fix all other comments.

With this one, ALWAYS_INLINE because I want the 'safe' compiled out when safe=0, so the fast route (without tilde) is as fast as it used to be.

Reply:
ok